ObjCVoiceCallKitQuickstart/ViewController.m

```diff
@@ -296,6 +305,8 @@ - (void)callInviteReceived:(TVOCallInvite *)callInvite {
 - (void)cancelledCallInviteReceived:(TVOCancelledCallInvite *)cancelledCallInvite {
     NSLog(@"cancelledCallInviteReceived:");
     
+    [self incomingPushHandled];
```

Comment:
👍

Reply:
What would happen in the race condition?
1. First Invite rejected. But haven't received cancelled Invite yet.
2. Second call invite received, displayed on callkit.
3. The received Cancelled Invite can be for the first call or for the second call.